datascience/maps.py

```diff
@@ -19,8 +14,27 @@ def draw_map(center, zoom, points=[], regions=[]):
     zoom -- zoom level
     points -- a sequence of MapPoints
     regions -- a sequence of MapRegions
+
+    TODO(sam): Add points and regions functionality
     """
-    # Some magic to get javascript to display the map
+    map = folium.Map(
+        location=center,
+        tiles='Stamen Toner',
```

Comment:
Note that I chose this arbitrarily — if you feel like there's a better map style to use let me know.